documentation/package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I think it would be beneficial to have a Playbook in [documentation > Playbook](https://apps.itsm.gov.bc.ca/bitbucket/projects/DES/repos/documentation/browse/PlayBooks)

describing how to add viewpress to a repository. It's probably a tool that generates the scaffolding, and maybe a few other steps, but It would help me and anyone else who needs to add this to their project :)

Reply:
We have docs:https://bcgov.sharepoint.com/:w:/r/teams/00845/_layouts/15/Doc.aspx?sourcedoc=%7B7651F4DA-DA2D-4661-9836-F3C23476A13B%7D&file=VUEPRESS%20DOCUMENTATION%20SITE%20INSTRUCTIONS.docx&action=default&mobileredirect=true&wdsle=0

Reply:
eww it's in sharepoint ;)